e2e-tests/IIoTPlatform-E2E-Tests/Standalone/PublishNodesFromFileTestTheory.cs

```diff
@@ -22,10 +23,24 @@ public class PublishNodesFromFileTestTheory {
         private readonly ITestOutputHelper _output;
         private readonly IIoTPlatformTestContext _context;
 
+        /// <summary>
+        /// Deployment for edgeHub and edgeAgent so called "base deployment"
+        /// </summary>
+        private readonly IoTHubEdgeBaseDeployment _ioTHubEdgeBaseDeployment;
+
+        /// <summary>
+        /// Deployment for OPC Publisher as standalone
+        /// </summary>
+        private readonly IoTHubPublisherDeployment _ioTHubPublisherDeployment;
```

Comment:
Please use interface IIoTHubEdgeDeployment instead of base class

Reply:
Done.